cookbooks/bcpc-hadoop/attributes/monitoring.rb

```diff
@@ -0,0 +1,109 @@
+default["bcpc"]["hadoop"]["zabbix"]["history_days"] = 1
+default["bcpc"]["hadoop"]["zabbix"]["trend_days"] = 15
+default["bcpc"]["hadoop"]["zabbix"]["cron_check_time"] = 240
+default["bcpc"]["hadoop"]["zabbix"]["mail_source"] = "zabbix.zbx_mail.sh.erb"
+default["bcpc"]["hadoop"]["zabbix"]["cookbook"] = nil 
+
+# Graphite queries which specify property to query and alarming trigger, severity(maps to zabbix's api -> trigger -> priority)
+# and owner who the trigger is routed to for resolution
+# Queries are structured as they appear in graphite dashboard
+# Refer Zabbix API(https://www.zabbix.com/documentation/2.2/manual/api/reference/) for the various attributes used
+# The following list is supposed to be a skeleton which should be extended/edited according to the deployment scenario
+# By default items, triggers actions are disabled and won't be monitored. To enable them, set "enable=true" in the json structure
+default["bcpc"]["hadoop"]["graphite"]["queries"] = {
```

Comment:
Can these be added to the germane recipe for the query? (E.g. could the namenode* recipes deploy the namenode queries as we do for the [copylogs](https://github.com/bloomberg/chef-bach/blob/88a319e7520f30442a4e1dfe3ef2e9016e5cc9ca/cookbooks/bcpc-hadoop/recipes/namenode_master.rb#L11-L19) parameters?)